_posts/2024-01-12-call-for-proposals-for-djangocon-us-2024-website.md

```diff
@@ -59,7 +61,7 @@ DjangoCon US will need:
 
 ## Full Website Deliverable
 
-* Delivery of the full website design
+* Delivery of the full website design and ideally a starter code base including layouts for blog posts and a few major landing pages such as the call for proposals, venue details, and code of conduct
```

Comment:
We should probably list the schedule list, schedule details, presenter details, and updated open graph previews since these pages have different layouts.

Reply:
Good call. Let me tweak this slightly

Reply:
Added a separate bulleted list in the deliverables section